Handlers/Phil.py

```diff
@@ -939,6 +939,22 @@
     process_includes=True,
 )
 
+# override default resolutionizer parameters
+phil_overrides = phil_scope.fetch(
+    source=parse(
+        """\
+xia2.settings {
+  resolution {
+    isigma = None
+    misigma = None
+  }
+}
+"""
+    )
+)
+
+master_phil = phil_scope.fetch(sources=[phil_overrides])
```

Comment:
I don't think it is necessary to assign to an intermediate `phil_overrides` variable here, you can directly assign to `master_phil` above at line 942 (and no need to rename the variable at line 17).